### ISSUE TITLE
swagger api documentation

### ISSUE THREAD

**Issue by microtechcloud** [microtechcloudsolutions/sms-gateway#7]
Document Service APIs.
The writer will use swagger so as to standardize the documentation across services

**Comment by OlivierJM** [comment]
Is the API already there, from what I can tell from the code its not yet right.

**Comment by microtechcloud** [comment]
> 
> 
> Is the API already there, from what I can tell from the code its not yet right.

yes . some services are not fully ready, but their interfaces are predefined.
however for some services that are near ready, or already done, we can have them documented.

**Comment by bennetthub500** [comment]
Hello - I’m a pro writer and editor transitioning to technical writing, and I’m looking to build my portfolio -  I could create an OpenAPI spec for you using Swagger (have been working on this recently) - I'd probably have everything I need if you could give me the endpoints and authorization.  Or are you doing 'spec-first'?   I could also write other docs, including marketing content, Readmes, user-facing content, etc.   Let me know if you'd like to talk.
